src/prebid.js

```diff
@@ -1004,7 +1004,7 @@ if (FEATURES.VIDEO) {
     const bids = fetchReceivedBids(markBidRequest, 'Improper use of markWinningBidAsUsed. It needs an adUnitCode or an adId to function.');
 
     if (bids.length > 0) {
-      bids[0].status = CONSTANTS.BID_STATUS.RENDERED;
```

Comment:
When would the `status` have to be set to `rendered` ? Is there a place in Core, or would that have to be handled by the Video Module ?

Reply:
it's part of the logic in `addWinningBid`: https://github.com/prebid/Prebid.js/blob/62200e3c43486e98a5b58bd2d59b0e687ab815a3/src/auctionManager.js#L46